models/topic_test.go

```diff
@@ -112,14 +111,16 @@ func TestTopicStateTransitionAllowed(t *testing.T) {
 // and not to any forbidden of invalid state
 func validateTransitionsToCreated(topic models.Topic) {
 	Convey("Then an allowed transition is successfully checked", func() {
-		So(topic.StateTransitionAllowed(models.StateTopicCompleted.String()), ShouldBeTrue)
+		So(topic.StateTransitionAllowed(models.StateCompleted.String()), ShouldBeTrue)
 	})
 	Convey("Then a forbidden transition to a valid state is not allowed", func() {
-		So(topic.StateTransitionAllowed(models.StateTopicPublished.String()), ShouldBeFalse)
+		So(topic.StateTransitionAllowed(models.StatePublished.String()), ShouldBeFalse)
 	})
 	Convey("Then a transition to an invalid state is not allowed", func() {
 		So(topic.StateTransitionAllowed("wrong"), ShouldBeFalse)
 	})
 }
 
 //!!! as the topic functionality grows, add more tests to cover added state transitions.
```

Comment:
Same as above - not sure these add value

Reply:
removed.